float64 confidence
string plate_number
